docs/Model_Assumptions.rst

```diff
@@ -139,6 +138,34 @@ Also, if you are aiming at very high minimal renewable factors, the simulation t
 
 The minimum renewable share is introduced to the energy system by `D2.constraint_minimal_renewable_share()` and a validation test is performed with `E4.minimal_renewable_share_test()`.
 
+Minimal degree of autonomcy constraint
+######################################
+
+The minimal degree of autonomy constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal degree of autonomy defined within the constraint. The degree of autonomy of the optimized energy system may also be higher than the minimal degree of autonomy. Please find the definition of here: :ref:`_kpi_degree_of_autonomy`
+
```

Comment:
Link doesn' seem to work though I have double checked and the label: **_kpi_degree_of_autonomy** is correct

Reply:
my above suggestion should fix it